keyboards/ghs/rar/readme.md

```diff
@@ -0,0 +1,15 @@
+# GHS.RAR
+
+![GHS.RAR](https://i.imgur.com/5n893ul.jpg)
+
+A seamless screwless 75% keyboard.
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32u4
+* Hardware Availability: [Gone Hacking Studio](https://gonehacking.studio)
```

Comment:
This domain doesn't seem to be registered.

Reply:
I think the website isn't live yet but I'm pretty sure the domain has been reserved already. I'll ask my colleagues about it.